ruma-events-macros/src/event_content.rs

```diff
@@ -56,6 +56,29 @@ impl Parse for EventMeta {
     }
 }
 
+struct MetaAttrs(Vec<EventMeta>);
+
+impl MetaAttrs {
+    fn is_custom(&self) -> bool {
+        self.0.iter().any(|a| a == &EventMeta::CustomRedacted)
+    }
+    fn get_event_type(&self) -> Option<&LitStr> {
+        self.0.iter().find_map(|a| a.get_event_type())
+    }
+}
+
+impl Parse for MetaAttrs {
+    fn parse(input: ParseStream) -> syn::Result<Self> {
```

Comment:
This looks to me like space-separated is accepted as well. Let's use `syn::punctuated::Punctuated::<EventMeta>::parse_terminated`.